qsignature/src/org/qcmg/sig/messages.properties

```diff
@@ -32,6 +32,7 @@ UNKNOWN_ERROR = An unknown error has occurred
 INPUT_FILE_READ_ERROR = Could not read input file {0}
 MIN_COVERAGE_OPTION_DESCRIPTION = Specifies the minimum coverage to be used in the comparison
 CUTOFF_OPTION_DESCRIPTION = Specifies the cutoff value to be used in the comparison
+MAX_CACHE_SIZE_OPTION_DESCRIPTION = Opt, number of files to store in the cache. Used by the Compare class. Set this when the qsignature vcf files to be compared have a large number of positions. Defaults to -1 (unlimited number of files in cache)
```

Comment:
It is great to put "Opt" at the begin, but could you use "Def=-1" as JP suggested.

Reply:
Sure